.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -45,3 +47,9 @@ pip3 install setuptools==59.6.0
 # dependency to python-Levenshtein
 # wheels are missing and have to built from scratch
 sudo apt-get update && sudo apt-get install -y build-essential python3-dev
+
+# make sure that git is setup properly
+git config --global --add safe.directory /workspaces/vehicle-app-python-template
```

Comment:
Fix for known issue :octocat:

Reply:
this won't work if someone uses the template with a different namen